js/math.js

```diff
@@ -0,0 +1,41 @@
+
+
+function sum(arr) {
```

Comment:
nitpick:
if you notice all the 4 functions (sum, deduct, divide, multiply) have the exact same functionality except the operator itself, what if we move the common code into a separate function?